index.css

```diff
@@ -748,7 +750,7 @@ footer {
   }
 }
 
-@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Your design is not very responsive when it comes to its mobile version. Kindly ensure that the content is not scrollable sideways

![image](https://user-images.githubusercontent.com/98453979/219085609-63168ce3-58d9-4e9d-8068-c7f9de362f0c.png)

- [ ] On your mobile version in the `speakers` section, the `more` button is missing. Kindly add it to match the original design.

![image](https://user-images.githubusercontent.com/98453979/219086569-d06b42bc-2e81-4826-be60-e7a8aec2cc70.png)